### PR TITLE
ttrpc: fix lint errors in build

### PR DESCRIPTION
**Pull request by austinvazquez**
Fixes needless borrow error from #165 

```
error: the borrowed expression implements the required traits
  --> build.rs:17:17
   |
17 |         .inputs(&["src/ttrpc.proto"])
   |                 ^^^^^^^^^^^^^^^^^^^^ help: change this to: `["src/ttrpc.proto"]`
   |
   = note: `-D clippy::needless-borrow` implied by `-D warnings`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow

error: could not compile `ttrpc` due to previous error
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:29: check] Error 101
Error: Process completed with exit code 2.
```

Signed-off-by: Austin Vazquez <macedonv@amazon.com>

### CODE REVIEW

**Comment by austinvazquez** [comment]
Looks like there is more here than meets the eye. Maybe best to leave this for a Rust dev who knows more than I.
https://github.com/rust-lang/rust-clippy/issues/9111

File: build.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ fn main() {
     protobuf_codegen::Codegen::new()
         .pure()
         .out_dir(out_dir)
-        .inputs(&["src/ttrpc.proto"])
+        .inputs("src/ttrpc.proto")
```
Comment (Tim-Zhang) [inline build.rs:17]: ["src/ttrpc.proto"] would be right

**Comment by Tim-Zhang** [comment]
@austinvazquez Instead of `"src/ttrpc.proto"`, `["src/ttrpc.proto"]` would be right. 
Would you like to reopen it or I will create a new pr tomorrow. 
Thanks anyway

**Comment by austinvazquez** [comment]
Thanks @Tim-Zhang; I'll give it a try.

**Review by Tim-Zhang — APPROVED** [review]
LGTM, thanks @austinvazquez

**Review by liubin — APPROVED** [review]

**Pull request merged**